locale/pt-br/texts.po

```diff
@@ -0,0 +1,734 @@
+#.  enter the information.
+#: src/screens/RegisterToken.js:33 src/screens/SendScanQRCode.js:69
+msgid "Manual info"
+msgstr "Digitar"
+
+#: src/screens/RegisterToken.js:48 src/screens/RegisterTokenManual.js:112
+msgid "REGISTER TOKEN"
+msgstr "REGISTRAR UM TOKEN"
+
+#: src/screens/RegisterToken.js:59
+msgid "Scan the token QR code"
+msgstr "Leia o QR code do token"
+
+#: src/screens/RegisterTokenManual.js:88
+msgid "You're going to register the following token:"
+msgstr "Você irá registrar o seguinte token:"
+
+#: src/screens/RegisterTokenManual.js:90
+msgid "Name: "
+msgstr "Nome: "
+
+#: src/screens/RegisterTokenManual.js:94
+msgid "Symbol: "
+msgstr "Símbolo: "
+
+#: src/screens/RegisterTokenManual.js:118
+msgid "Configuration string"
+msgstr "Configuração"
+
+#: src/screens/ResetWallet.js:75
+msgid "RESET WALLET"
+msgstr "RESETAR WALLET"
+
+#: src/screens/ResetWallet.js:79
+msgid "Are you sure?"
+msgstr "Você tem certeza?"
+
+#: src/screens/ResetWallet.js:82
+msgid ""
+"If you reset your wallet, **all data will be deleted**, and you will **lose "
+"access to your tokens**."
+msgstr ""
+"Se você resetar sua wallet, **todos os seus dados serão apagados**, e você "
+"irá **perder acesso aos seus tokens**."
+
+#: src/screens/ResetWallet.js:84
+msgid ""
+"To recover access to your tokens, you will need to import your seed words "
+"again."
+msgstr ""
+"Para recuperar acesso aos seus tokens, você irá precisar importar suas "
+"palavras novamente."
+
+#: src/screens/ResetWallet.js:93
+msgid ""
+"I want to reset my wallet, and I acknowledge that **all data will be wiped "
+"out**."
+msgstr ""
+"Eu quero resetar minha wallet, e eu entendo que **todos os dados serão "
+"deletados**."
+
+#: src/screens/ResetWallet.js:102
+msgid "Reset Wallet"
+msgstr "Resetar Wallet"
+
+#: src/screens/Security.js:73
+msgid "No biometry supported"
+msgstr "Nenhuma biometria suportada"
+
+#: src/screens/Security.js:73
+#, javascript-format
+msgid "Use ${ this.supportedBiometry }"
+msgstr "Usar ${ this.supportedBiometry }"
+
+#: src/screens/Security.js:77
+msgid "SECURITY"
+msgstr "SEGURANÇA"
+
+#: src/screens/Security.js:96
+msgid "Change PIN"
+msgstr "Mudar PIN"
+
+#: src/screens/Security.js:100
+msgid "Lock wallet"
+msgstr "Bloquear a wallet"
+
+#: src/screens/SendAddressInput.js:59
+msgid "Address to send"
+msgstr "Endereço para enviar"
+
+#: src/screens/SendAmountInput.js:94
+msgid "Insufficient funds"
+msgstr "Saldo insuficiente"
+
+#: src/screens/SendAmountInput.js:118 src/screens/SendConfirmScreen.js:125
+#, javascript-format
+msgid "${ amountAndToken } available"
+msgid_plural "${ amountAndToken } available"
+msgstr[0] "${ amountAndToken } disponível"
+msgstr[1] "${ amountAndToken } disponíveis"
+
+#: src/screens/SendAmountInput.js:126 src/screens/SendConfirmScreen.js:133
+msgid "SEND ${ tokenNameUpperCase }"
+msgstr "ENVIAR ${ tokenNameUpperCase }"
+
+#.  show loading modal
+#.  eslint-disable-next-line react/jsx-indent
```

Comment:
Why is this needed?

Reply:
The same here. I'm not sure why it was collected as a note for translators.